internal/commands/list_buildpack_registries_test.go

```diff
@@ -76,11 +76,21 @@ func testListBuildpackRegistriesCommand(t *testing.T, when spec.G, it spec.S) {
 			h.AssertContains(t, outBuf.String(), "public registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/public-registry")
 
-			h.AssertContains(t, outBuf.String(), "private registry")
+			h.AssertContains(t, outBuf.String(), "* private registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/private-registry")
 
 			h.AssertContains(t, outBuf.String(), "personal registry")
 			h.AssertContains(t, outBuf.String(), "https://github.com/buildpacks/personal-registry")
 		})
+
+		it("should ALWAYS list an official default registry", func() {
```

Comment:
This test says "ALWAYS" but it's only testing when there are not other registries.

Can we have two tests? One when no registries are added and another for when 1 or more registries are added.

FWIW: I agree that official should always be displayed. It currently doesn't _always_ display. It only display when there are no others:

```
$ ./out/pack list-buildpack-registries -v
  my-registry https://github.com/buildpacks/my-buildpack
  my-registry2 https://github.com/buildpacks/my-buildpack
* my-registry3 https://github.com/buildpacks/my-buildpack
```

Reply:
My approach was to treat the `official` registry as a fallback in case there weren't any registries.  Similiar to what we're already doing with https://github.com/buildpacks/pack/blob/4c29bf83e989fdf0843183a3b6c2d21d1ce24f6d/internal/config/config.go#L105.   But maybe we should always include it with the others.  I need to think about this some more.